function-support.html

```diff
@@ -593,7 +612,7 @@ <h2 id="environments">Environments</h2>
 	
         <p>Acceptable line separators include: <code>\\</code>, <code>\cr</code>, and <code>\cr[<em>distance</em>]</code>. <em>Distance</em> can be written with any of the <a href="#units">KaTeX units</a>.</p>

```

Comment:
I know you like `\cr` because it's less likely to cause issues, but it still seems worth mentioning `\\[distance]` too.

Reply:
I promise that I'll put `\\[distance]` back in as soon as we fix auto-render.  I'm pretty sure that we'll make the next release.